messages/observations/heading.json

```diff
@@ -24,6 +24,8 @@
             "title": "Type of sensor",
             "enum": [
                 "Magnetic",
+                "Gyro",
+                "IMU",
                 "True"
```

Comment:
In the NMEA0183 spec, there is only `Magnetic` and `True`:
https://gpsd.gitlab.io/gpsd/NMEA.html#_hdm_heading_magnetic
https://gpsd.gitlab.io/gpsd/NMEA.html#_hdt_heading_true

Is it generally considered that a gyro produce a "True" heading? How do we in that case decide on what to put when constructing this message? 

Perhaps "True" is superfluous?